### DISCUSSION TITLE
Make fzf the autocomplete handler for `cd`

### DISCUSSION THREAD

**Discussion by DannyBen** [junegunn/fzf#3019] [Q&A]
I know there are several techniques and a built in functionality to *sort of* complete paths when typing `cd <tab>` - like the `cd **<tab>` and <kbd>Alt</kbd>+<kbd>C</kbd>.

I do not wish to use the `**` (or any) prefix, and I do not want to see the fzf menu. I just want it suggest (as if using `--filter`) fuzzy-matched directories **in addition** to the native `cd` autocomplete, using the native shell completion.

In other words, I wish to implement this:

```bash
$ cd smd<tab>
/some/directory        # highest fzf score
/some/other/directory  # next fzf match
/opt/smd               # since /opt is in my CDPATH (offered by native completion)
./smd                  # offered by native completion
```

I have spent hours searching, any hint is appreciated.
